backend/de.metas.adempiere.adempiere/base/src/test/java/de/metas/impexp/format/ImpFormatColumnTest.java

```diff
@@ -0,0 +1,71 @@
+package de.metas.impexp.format;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.sql.Timestamp;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ImpFormatColumnTest
+{
+
+	private ImpFormatColumn impFormatColumn;
+
+	@BeforeEach
+	void setUp()
+	{
+		impFormatColumn = ImpFormatColumn.builder()
+				.columnName("testColumn")
+				.dataType(ImpFormatColumnDataType.Date)
+				.dataFormat("dd.MM.yy") // Using 2-digit year format
+				.build();
+	}
+
+	@Test
+	void testParseCellValue_TwoDigitYear_ShouldParseTo2025() throws Exception
+	{
+		final String inputDate = "21.03.25"; // 2-digit year
+		final Timestamp expectedTimestamp = Timestamp.valueOf("2025-03-21 00:00:00");
+
+		final Object result = impFormatColumn.parseCellValue(inputDate);
+
+		assertNotNull(result);
+		assertTrue(result instanceof Timestamp);
+		assertEquals(expectedTimestamp, result);
```

Comment:
@riadmen @metas-ts  
* next time pls use assertJ, the whole test could be written in one sentence
* assertNotNull is redundant if u are checking `assertTrue(result instanceof Timestamp)`

e.g.
```
assertThat(impFormatColumn.parseCellValue("21.03.25"))
.isInstanceOf(TImestamp.class)
.isEqualTo(Timestamp.valueOf("2025-03-21 00:00:00"))
```